features/fixtures/mazerunner/jvm-scenarios/src/main/java/com/bugsnag/android/mazerunner/scenarios/DiscardBigEventsScenario.kt

```diff
@@ -34,9 +34,5 @@ internal class DiscardBigEventsScenario(
         super.startScenario()
         Bugsnag.markLaunchCompleted()
         Bugsnag.notify(MyThrowable("DiscardBigEventsScenario"))
-
-        waitForEventFile()
```

Comment:
If we're not going to use `waitForEventFile` in this scenario - we should also remove the method?